js/jquery.keyboard.js

```diff
@@ -753,6 +753,11 @@ http://www.opensource.org/licenses/mit-license.php
 					k1 = k >= keyCodes.A && k <= keyCodes.Z,
 					k2 = k >= keyCodes.a && k <= keyCodes.z,
 					str = base.last.key = String.fromCharCode(k);
+				// check, that keypress wasn't rise by functional key
+				// space is first typing symbol in UTF8 table
+				if (k < keyCodes.space){ //see #549
```

Comment:
It duplicate a `keyCode` or have zero in FF for function keys and `charCode` for others.
And it represent `charCode` in Opera, Chrom, IE.

There is snippet, its helped me understand how keypress works in different browsers:
[https://jsfiddle.net/VikEdit/9L3Lva82/](https://jsfiddle.net/VikEdit/9L3Lva82/)